Index.html

```diff
@@ -1,269 +1,176 @@
-                    <i class="fa fa-solid fa-arrow-left"></i>
-                    previous project
-                </button>
-                <button class="buttons__inverse nav__pop__next nav__pop__down">
-                    next project
-                    <i class="fa fa-light fa-arrow-right"></i>
-                </button>
-            </div>
-        </section>
-    </main>
-
-    <footer class="footer">
-        <button class="buttons footer__button">
-            <h4 class="footer__button__text">Get my resume</h4>
-            <img class="footer__button__image" src="./Contents/Resources/Icons/download_Resume.svg" alt="download my resume">
-        </button>
-
-        <ul class="socials">
-            <li href=""><img src="./Contents/Resources/Icons/ic_stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_github.svg" alt="go to my github" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_twitter.svg" alt="go to my twitter" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_linkedin.svg" alt="go to my linkendin" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/ic_medium.svg" alt="go to my M" class="social"></li>
-            <li href=""><img src="./Contents/Resources/Icons/fingers.svg" alt="go to my M" class="social"></li>
+      </li>
+      <li>Frameworks
+        <ul id="frameworks">
+          <li>Bootstrap</li>
+          <li>Django</li>
+          <li>Rest</li>
+          <li>Celery</li>
+          <li>Postgres</li>
+        </ul>
+      </li>
+      <li>Skills
+        <ul id="skills">
+          <li>Problem solving</li>
+          <li>Agile</li>
+          <li>GitHub</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
         </ul>
-    </footer>
+      </li>
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
```

Comment:
- [ ] As pointed out by the previous reviewer, kindly design the form according to the given design. 🙏 

> |     | Given template                                | Your Design                                   |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="120" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/236432890-a52e9b7f-99d2-4091-9d0b-f69e25daf1c1.png"> |<img width="270" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/236432547-71659753-17fb-42cc-9516-35e8b0a5a15d.png"> |

Reply:
All of you are stupid, do you understand it is a desktop version branch and I could not work on the mobile version here? and still, review the mobile version????????

Reply:
### Hello @mm-pazouki 👋 👋,
I understand your frustration, but please remember to remain respectful when communicating with others. Please, keep the conversation professional and respectful. 🙏 

Note that the milestone requires the webpage to be fully responsive. This means that it should work seamlessly on both desktop and mobile devices. Therefore, it is important to address any issues related to the mobile version as well. 😎😎